### PR TITLE
intercal 0.32

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
```
  /bin/sh ./buildaux/ylwrap `test -f 'lexer.l' || echo './'`lexer.l lex.yy.c lexer.c -- flex  
  clang: error: unknown argument: '-fno-toplevel-reorder'
  make[1]: *** [parser.o] Error 1
  make[1]: *** Waiting for unfinished jobs....
  make: *** [install] Error 2

```

File: Formula/i/intercal.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Intercal < Formula
   desc "Esoteric, parody programming language"
   homepage "http://catb.org/~esr/intercal/"
-  url "http://catb.org/~esr/intercal/intercal-0.31.tar.gz"
-  sha256 "93d842b81ecdc82b352beb463fbf688749b0c04445388a999667e1958bba4ffc"
+  url "http://catb.org/~esr/intercal/intercal-0.32.tar.gz"
```
Comment (colindean) [inline Formula/i/intercal.rb:4]: **suggestion:** Could replace the non-working homepage with its source repo: https://gitlab.com/esr/intercal, perhaps also replace the `url`, too.

File: Formula/i/intercal.rb [hunk 2]
```diff
@@ -36,29 +36,38 @@ class Intercal < Formula
...
 
   test do
     (testpath/"lib").mkpath
     (testpath/"test").mkpath
+
     cp pkgshare/"pit/beer.i", "test"
     cd "test" do
       system bin/"ick", "beer.i"
-      system "./beer"
+      assert_equal <<~EOS, shell_output("./beer 2>&1", 240)
```
Comment (chenrui333) [inline Formula/i/intercal.rb:66]: the build might just not work on macos

Comment (chenrui333) [reply Formula/i/intercal.rb:66]:
```
Take one down, pass it around,
 0 bottles of beer on the wall.

/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
Minitest::Assertion: Expected: 240
  Actual: 0
1. raise
2. ignore
3. backtrace
4. irb
5. shell
Choose an action: 1
==> Temporary files retained at:
/tmp/intercal-test-20240303-15779-6wrt4h
Error: intercal: failed
An exception occurred within a child process:
  Minitest::Assertion: Expected: 240
  Actual: 0
```

**Comment by chenrui333** [comment]
raised an upstream issue on macos build failure, https://gitlab.com/esr/intercal/-/issues/5

File: Formula/i/intercal.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/i/intercal.rb]: yeah, I think we can still use the catb.org url, there are many other software on the same site.

**Review by chenrui333 — APPROVED** [review]

File: Formula/i/intercal.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
   desc "Esoteric, parody programming language"
   homepage "http://catb.org/~esr/intercal/"
-  url "http://catb.org/~esr/intercal/intercal-0.31.tar.gz"
-  sha256 "93d842b81ecdc82b352beb463fbf688749b0c04445388a999667e1958bba4ffc"
-  license "GPL-2.0"
+  url "http://catb.org/~esr/intercal/intercal-0.32.tar.gz"
+  sha256 "a3c1e0a7ac591f78b7f5af42fdc6ae7d2a02981b7862c0320d3537beb54dfd78"
+  license "GPL-2.0-or-later"
 
   # The latest version tags in the Git repository are `0.31` (2019-06-12) and
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/i/intercal.rb]: okay, just added 0.32 release note, let me know if that works.

[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/i/intercal.rb]: also logged an issue to ask upstream to remove those seemingly wrong tags
https://gitlab.com/esr/intercal/-/issues/6

**Comment by iMichka** [comment]
I'm wondering if it is OK to drop macOS support so quickly. We could wait a few days for an answer from upstream. This has a low download count so the impact is low, but I think we should not too quickly drop macOS.

**Comment by chenrui333** [comment]
> I'm wondering if it is OK to drop macOS support so quickly. We could wait a few days for an answer from upstream. This has a low download count so the impact is low, but I think we should not too quickly drop macOS.

I doubt upstream would answer my call as it happens for his other formulae (but we can hold it for couple of days though, works fine for me)

**Comment by chenrui333** [comment]
https://github.com/Homebrew/homebrew-core/pull/245726

File: Formula/i/intercal.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/i/intercal.rb]: - https://github.com/Homebrew/homebrew-core/pull/260091

**Pull request closed without merging**